honda_civic_touring_2016_can.dbc

```diff
@@ -198,6 +198,8 @@ BO_ 513 GAS_SENSOR: 5 INTERCEPTOR
  SG_ CHECKSUM : 35|4@0+ (1,0) [0|3] "" NEO
 
 BO_ 545 XXX_10: 6 XXX
+ SG_ ECON_ON_2 : 37|2@0+ (1,0) [0|3] "" NEO

```

Comment:
2 bits? What value corresponds to eco mode on? I would add a value table at the end of dbc to say what 0,1,2,3 mean.